report-module.js

```diff
@@ -0,0 +1,30 @@
+'use-strict';
+
+const email_log = require('./email_log.json');
+
+const bugReportModule = function () {
+  const getBugReport = (dataLog) => {
+    console.log(dataLog.emails.reduce((customers, currentCustomer) => {
```

Comment:
It's best not to do work in a console.log. Instead do all the work and set it to a variable and console.log that variable.